src/box/lua/stat.c

```diff
@@ -271,6 +302,20 @@ lbox_stat_net_call(struct lua_State *L)
 	lua_rawset(L, -3);
 	lua_pop(L, 1);
 
+	lua_pushstring(L, "STREAMS");
+	lua_rawget(L, -2);
+	lua_pushstring(L, "current");
+	lua_pushnumber(L, iproto_stream_count());
+	lua_rawset(L, -3);
+	iproto_stream_rmean_foreach(set_stat_item, set_iproto_thread_request_stat_item, L);
+	lua_pushstring(L, "requests");
+	lua_rawget(L, -2);
+	lua_pushstring(L, "current");
+	lua_pushnumber(L, iproto_stream_request_count());
+	lua_rawset(L, -3);
+	lua_pop(L, 1);
+	lua_pop(L, 1);
+
```

Comment:
STREAMS.requests looks inconsistent with the rest of the box.stat.net() output (e.g. there's no CONNECTIONS.requests and at the same time there is REQUESTS). Does the user really need to know the count of streamlined messages? Why?

I assume the user wants to know how many messages are processed in parallel, that's what REQUESTS is for. Does REQUESTS include queued stream-lined requests? I think it shouldn't. Maybe, there should be a separate counter for queued requests waiting to be processed on stream?